### PR TITLE
ecs-app Terraform Module e2e tests

### PR DESCRIPTION
**Pull request by igorkotof**
Test: https://github.com/hazelops/terraform-aws-ecs-app/actions/runs/3504773039

### CODE REVIEW

File: .github/workflows/run.tf-examples-tests.yml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  IZE_LOG_LEVEL: debug
+  IZE_PREFER_RUNTIME: native
+  IZE_PLAIN_TEXT: true
+  IZE_VERSION: 1.1.9
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - core-448
```
Comment (DimmKirr) [inline .github/workflows/run.tf-examples-tests.yml]: Should this be changed?

File: .github/workflows/run.tf-examples-tests.yml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+      - name: IZE gen tfenv 
+        run: ize gen tfenv
+      
+      - name: Copy generated files
+        run: |
+          cp -R .ize/env/${{ env.ENV }}/*.* examples/${{ github.job }}/
+#debug     cat examples/${{ github.job }}/backend.tf
+#          ls -la examples/${{ github.job }}
+#          ls -la .ize/env/${{ env.ENV }}
```
Comment (DimmKirr) [inline .github/workflows/run.tf-examples-tests.yml]:
```suggestion
```

File: .github/workflows/run.tf-examples-tests.yml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+      - name: Go TF Test
+        run: |
+          NAME=${{ github.job }}
+          cd test/src
+          go mod tidy
+          go test -v -timeout 60m -run TestExamplesWebProxy
+
+  complete-worker:
+    name: complete-worker
```
Comment (DimmKirr) [inline .github/workflows/run.tf-examples-tests.yml]: Not necessary as it's the same as the job id
```suggestion
```

File: .github/workflows/run.tf-examples-tests.yml [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+      - name: IZE gen tfenv 
+        run: ize gen tfenv
+      
+      - name: Copy generated files
+        run: |
+          cp -R .ize/env/${{ env.ENV }}/*.* examples/${{ github.job }}/
+#debug     cat examples/${{ github.job }}/backend.tf
+#          ls -la examples/${{ github.job }}
+#          ls -la .ize/env/${{ env.ENV }}
```
Comment (DimmKirr) [inline .github/workflows/run.tf-examples-tests.yml]:
```suggestion
```

File: .github/workflows/run.tf-examples-tests.yml [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+      - name: Go TF Test
+        run: |
+          NAME=${{ github.job }}
+          cd test/src
+          go mod tidy
+          go test -v -timeout 60m -run TestExamplesCompleteWorker
+
+
+
+
```
Comment (DimmKirr) [inline .github/workflows/run.tf-examples-tests.yml]: Hyper new-line :-D
```suggestion
```

File: .github/workflows/run.tf-examples-tests.yml [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+      - name: IZE gen tfenv 
+        run: ize gen tfenv
+      
+      - name: Copy generated files
+        run: |
+          cp -R .ize/env/${{ env.ENV }}/*.* examples/${{ github.job }}/
+#debug     cat examples/${{ github.job }}/backend.tf
+#          ls -la examples/${{ github.job }}
+#          ls -la .ize/env/${{ env.ENV }}
```
Comment (DimmKirr) [inline .github/workflows/run.tf-examples-tests.yml]:
```suggestion
```

**Review by DimmKirr — APPROVED** [review]
Looks good, 
Please address the requests and feel free to merge!

File: .github/workflows/run.tf-examples-tests.yml (hunk 1 above)
Comment (igorkotof) [reply .github/workflows/run.tf-examples-tests.yml]: yes

**Pull request merged**